src/views/JobView.vue

```
@@ -0,0 +1,101 @@
+<script setup lang="ts">
+import { reactive, onMounted } from "vue"
+import axios from "axios"
+import { useRoute, RouterLink } from "vue-router"
+import type { Job } from "@/types/global"
+
+const route = useRoute()
+
+const state = reactive<{
+    job: Job | {};
```

ESLint

[error] The `{}` ("empty object") type allows any non-nullish value, including literals like `0` and `""`.
- If that's what you want, disable this lint rule with an inline comment or configure the 'allowObjectTypes' rule option.
- If you want a type meaning "any object", you probably want `object` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.

(@typescript-eslint/no-empty-object-type)